scripts/lib/libvirt/admin-config.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+
+import argparse
+from string import Template
+import sys
+
+import libvirt
+import libvirt_helpers
+
+conn = libvirt.open("qemu:///system")
+if not conn:
+    print("Failed to open connection to the hypervisor")
+    sys.exit(1)
+
+parser = argparse.ArgumentParser(description="Create Admin Node Config.")
```

Comment:
This `argparse` code should be in `main()`, so that it doesn't get executed if the file is being used as a module.